website/docs/faqs/Environments/delete-environment-job.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to delete a job or environment in dbt Cloud?
+description: "How to delete a job or environment"
+sidebar_label: 'Delete a job or environment'
+id: delete-environment-job
+--- 
+
+
+To delete an environment or job in dbt Cloud, you must have a `developer` [license](/docs/collaborate/manage-access/seats-and-users) and have the necessary [access permissions](/docs/collaborate/manage-access/about-access). 
```

Comment:
This could be confusing to folks who have any other license type (or even trial users). Would it make sense to only reference the user permissions required to delete either?